app/routes.js

```diff
@@ -71,11 +71,14 @@ export const routes = (authProvider) => {
     const { organisationId } = req.params;
     const response = await postAddUser({ organisationId, data: req.body, accessToken: extractAccessToken({ req, tokenType: 'access' }) });
     if (response.success) {
-      res.redirect(`/organisations/${organisationId}/adduser/confirmation?userAdded=${response.userAdded}`);
-    } else res.send('Error adding user');
-    // TODO: Implement with errors
-    // const context = await getAddUserPageErrorContext({ validationErrors: response (etc) });
-    //   return res.render('pages/adduser/template', context);
+      return res.redirect(`/organisations/${organisationId}/adduser/confirmation?userAdded=${response.userAdded}`);
+    }
+    const context = await getAddUserPageErrorContext({
+      validationErrors: response.errors,
+      organisationId,
+      accessToken: extractAccessToken({ req, tokenType: 'access' }),
```

Comment:
We are extracting the accessToken twice in this route. Let's have a const at the start for accessToken and then we can pass that instead.